runtime/Cargo.toml

```diff
@@ -43,3 +45,6 @@ name = "solana_runtime"
 [dev-dependencies]
 assert_matches = "1.3.0"
 solana-noop-program = { path = "../programs/noop", version = "0.24.0" }
+
+[build-dependencies]
```

Comment:
Due to the same restriction of [this](https://github.com/solana-labs/solana/pull/8012/files#r374398085), we're also forced to add this magic to use frozen_abi in addition to the usual of adding to `[dependencies]`.